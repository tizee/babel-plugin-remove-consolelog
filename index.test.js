const babel = require('@babel/core');
const removeConsole = require('./index');

const testCode = `
  console.log("hello");
  console.log("hello");
  console.log("hello");
  console.log("hello");
  function foo(){
    console.log("hello");
    console.log("hello");
    console.log("hello");
    console.log("hello");
  }
  const bar = ()=> {
    console.log("hello");
  }
  const fooBar = console.log;
  fooBar();
`;

it('Test removeConsole', () => {
  const { code } = babel.transform(testCode, {
    plugins: [removeConsole],
  });
  expect(code).toMatchSnapshot();
});
