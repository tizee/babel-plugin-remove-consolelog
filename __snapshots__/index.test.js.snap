// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test removeConsole 1`] = `
"function foo() {}

const bar = () => {};

const fooBar = console.log;
fooBar();"
`;
